ush/valid_param_vals.yaml

```diff
@@ -78,4 +78,4 @@ valid_vals_DO_AQM_SAVE_AIRNOW_HIST: [True, False]
 valid_vals_COLDSTART: [True, False]
 valid_vals_VX_FIELDS: [ "APCP", "ASNOW", "REFC", "RETOP", "ADPSFC", "ADPUPA" ]
 valid_vals_VX_APCP_ACCUMS_HRS: [ 1, 3, 6, 24 ]
-valid_vals_VX_ASNOW_ACCUMS_HRS: [ 6, 24 ]
+valid_vals_VX_ASNOW_ACCUMS_HRS: [ 6, 12, 18, 24 ]
```

Comment:
I don't understand when 12 or 18 would ever be used here; wouldn't that require that we have obs files of 12- or 18-hour snowfall accumulation?

Reply:
For ASNOW, this PR changes things so that 6-hourly accumulations are used to build the longer accumulations (if asked for), both for the obs and the forecasts.  So any multiple of 6hr will work.  We can theoretically go beyond 24, too (30, 36, ...), but I guess that's not ever done in practice.